### ISSUE TITLE
how to remove bullet points from 'simple' list using html.xsl

### ISSUE THREAD

**Issue by CJohnno77** [TEIC/TEI#2211]
I am using `html.xsl` on an `.xml` file which has lists of type 'simple': eg. `<list type="simple">`

The items of these lists are coming out as `<li class="item">` in the resulting HTML, which display with bullet points.

Is there a way to get `simple` lists to not have the bullet points?  Is there a parameter I can set in the XSL?

**Comment by martindholmes** [comment]
The current TEI recommendation is for `@rend="simple"` rather than `@type` (https://tei-c.org/release/doc/tei-p5-doc/en/html/CO.html#COLI). If you change that, does it work?

**Comment by CJohnno77** [comment]
> 
> 
> The current TEI recommendation is for `@rend="simple"` rather than `@type` (https://tei-c.org/release/doc/tei-p5-doc/en/html/CO.html#COLI). If you change that, does it work?

No, it didn't work - I still get the `<ul class="simple">` in the HTML which displays the bullet points.

**Comment by sydb** [comment]
So, if I understand correctly so far, you (@CJohnno77) want to produce on output an HTML list that does not have bullets (and presumably does not have numbers, either).

My instinct is that, although @martindholmes is correct about the encoding, it is a red herring. Since `"simple"` is not a value of the `@class` of `<html:ul>` that the CSS recognizes, it does not matter whether or not you successfully get it into the HTML or not. Seems to me the question boils down to “How do I tell the CSS not to use bullets for `<ul class="simple">`?”. I am _far_ from a CSS expert, but I think
~~~css
        ul.simple {
          list-style-type: none;
          }
~~~
will do the trick. How you get that in there is another story. If you are already set, great. If not, just say so. (There are parameters named `cssHook` and `cssInlineFile`,  but I have never used them.)

**Comment by martindholmes** [comment]
If "simple" isn't supported in the CSS, I think it should be.

**Comment by sydb** [comment]
I do not disagree, but if `"simple"` _means_ “use bullets, not numbers” then it already is supported in the CSS. 😄

**Comment by martindholmes** [comment]
No, "simple" means use no markers. "bulleted" means use bullets. Suggestion list is here:

https://tei-c.org/release/doc/tei-p5-doc/en/html/CO.html#COLI

**Comment by sydb** [comment]
Ah. I see.

**Comment by CJohnno77** [comment]
> How you get that in there is another story. If you are already set, great. If not, just say so. (There are parameters named `cssHook` and `cssInlineFile`, but I have never used them.)

I could add CSS to the resulting HTML to turn off the bullets for `ul.simple`, but I would prefer to use a parameter in the XSL, so I will look into `cssHook` and `cssInlineFile`. 

Is there a CSS file I should be using when I use `html.xsl`, or should I make one myself?

**Comment by lb42** [comment]
I would recommend avoiding all this folderol by using ceteicean

Reluctantly using Outlook for Android<https://aka.ms/ghei36>
________________________________
